modules/ROOT/pages/troubleshoot-service-binding-issues.adoc

```diff
@@ -0,0 +1,115 @@
+= Troubleshooting Service Binding Issues
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, service binding issues, troubleshooting
+
+After you create the API, you then bind your service with the API. This section provides possible causes for issues that might occue when you bind your service, and describes how to troubleshoot these errors:
+
+* <<dns-name-error,DNS Name Error>>
+* <<references non-existent-instance-error,References non-existent instance error>>
+* <<missing-entitlements-in-license,Missing entitlements in license>>
+
+[[dns-name-error]]
+== DNS Name Error
+
+If you provide a DNS name that does not comply with the DNS subdomain standard format, an error occurs and the binding fails.
+
+=== Cause
+
+This issue occurs if you provided a DNS name that does not follow the DNS subdomain standards.
+
+=== Diagnose
+
+To diagnose this issue, run one of the following commands:
++
+`$ ./service-mesh api binding list` Or `$ ./service-mesh adapter binding list`.
++
+The output of this command displays a Failed status of the binding:
```

Comment:
The output of which command? Both? Or only the second one above? Clarify.

Reply:
Here is my best attempt, but please be sure it is not misleading:
```suggestion
The output displays that binding failed:
```

Reply:
Done! Thanks!